src/redis-benchmark.c

```diff
@@ -1020,7 +1021,7 @@ static benchmarkThread *createBenchmarkThread(int index) {
     if (thread == NULL) return NULL;
     thread->index = index;
     thread->el = aeCreateEventLoop(1024*10);
-    aeCreateTimeEvent(thread->el,1,showThroughput,NULL,NULL);
+    aeCreateTimeEvent(thread->el,1,showThroughput,(void *)(uintptr_t)(index),NULL);
```

Comment:
As long as the thread id is signed the`int` should be castted to `intptr_t` and not `uintptr_t`.
```suggestion
    aeCreateTimeEvent(thread->el,1,showThroughput,(void *)(intptr_t)index,NULL);
```